istio/gob-minikube/README.md

```diff
@@ -0,0 +1,49 @@
+# Gob's microservice
+
+Run [Gob](https://github.com/linkerd/linkerd-examples/tree/master/gob)
+in minikube with Istio.
+
+## Istio-Envoy
+Follow all of the steps in the
+[istio installation guide](https://istio.io/docs/tasks/installing-istio.html)
+to install the istio components.
+
+```
+# Add Ingress Resource
+kubectl apply -f gob-ingress.yaml
+
+# Deploy Gob
+kubectl apply -f <(istioctl kube-inject -f gen.yaml)
+kubectl apply -f <(istioctl kube-inject -f word.yaml)
+kubectl apply -f <(istioctl kube-inject -f web.yaml)
+```
+
+Test that it works:
+```
+LB=$(minikube ip):$(kubectl get svc istio-ingress -o jsonpath="{.spec.ports[0].nodePort}")
+open http://$LB/gob
+```
+
+## Istio-Linkerd
+
+Assumes you've gone through the first four steps of the
+[Istio installation guide](https://istio.io/docs/tasks/installing-istio.html).
+
+```
+# Deploy istio-pilot, istio-mixer and egress
+kubectl apply -f ../mixer-pilot.yml -f ../istio-egress.yml
+
+# Add grpc-specific linkerds
+kubectl apply -f ../istio-daemonset-grpc.yml
+
+# Add http ingress linkerd
+kubectl apply -f ../istio-ingress.yml
+
+# Add Ingress Resource
+kubectl apply -f gob-ingress.yaml
```

Comment:
Super minor, but how does `Add http ingress linkerd` relate to `Add Ingress Resource`? Maybe this would be clearer as `Add linkerd http ingress controller` and `Add gob http ingress resource`, if that's actually a correct description of what these files do (ianae)?

Reply:
yeah one is a linkerd being used as an ingress controller, and the other is the k8s ingress resource to direct incoming traffic to gob. will update the comments!